modules/addon/versions.tf

```
@@ -4,11 +4,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 4.47"
+      version = "~> 5.0"
     }
     helm = {
       source  = "hashicorp/helm"
-      version = ">= 2.9"
+      version = "~> 2.0"
```

```suggestion
      version = "~> 2.9"
```
(more compatible with the previous `>=` operator)